api/v2.0/swagger.yaml

```diff
@@ -7252,6 +7252,10 @@ definitions:
         type: string
         description: 'The ID of the tag retention policy for the project'
         x-nullable: true
+      proxy_speed_kb:
+        type: string
+        description: 'The bandwidth limit of proxy cache, in Kbps (kilobits per second)'
```

Comment:
It limits the communication from Harbor to the upstream registry, not the client to the Harbor.